wallets/ethereum-wallet-mock/src/index.ts

```diff
@@ -1,3 +1,3 @@
-export * from './ethereum-wallet-mock'
+export * from './EthereumWalletMock'
```

Comment:
EthWalletMock?

Reply:
the same as the class name the file includes